packages/playground/src/services.ts

```diff
@@ -13,6 +13,29 @@ import "./style.css";
 
 export function attachServices(host: BrowserHost) {
   monaco.languages.register({ id: "cadl" });
+  monaco.languages.setLanguageConfiguration("cadl", {
```

Comment:
Is it possible to load this from the same JSON so we don't have to make changes twice?

Reply:
LOL, there's always some rage-inducing, minor impendence mismatch between monaco and VS Code. I suppose we could have shared data and generate the .json file from it too. Don't block on this. If it's hard, it's probably not worth it since this data almost never changes. Maybe just a comment to update the other location in both.

Reply:
We don't have to load it from JSON right? Could just export a constant from the LS, and a function could mutate the structure for use in the playground fairly trivially..

Reply:
For VS Code, AFAIK, it has to be a .json file, but we could generate it in the build, I guess.

Reply:
It's possible there's a way to provide it to VS Code without using the .json file, I haven't researched.

Reply:
I think it being JSON linked from package.json will let VS Code light some things up while the extension is still loading, much like tmlanguage.

Reply:
I'll check on that. I was trying to find a jsonschema for that file in vscode and couldn't find last time.

Reply:
using the same one works, from what I can see it has to be json however. Updated to include the configuration from the compiler server.

Question/issues
- There is no typing for the configuration apart from the monaco editor types but we shouldn;t include those in the compiler. So either we don't do typing which is the same we basically had when it was in json or we copy over the types which ends up being quite a lot.
- Should the generated language-configuration.json be commited or not?

Reply:
I would prefer not to commit the file. The cast is OK.